test/Strings/rlexe.xml

```diff
@@ -230,4 +230,11 @@
       <tags>BugFix</tags>
     </default>
   </test>
+  <test>
+    <default>
+      <files>unicode_toUpperCase_toLowerCase.js</files>
+      <compile-flags>-args summary -endargs</compile-flags>
+      <tags>exclude_win7</tags>
```

Comment:
@kunalspathak thanks for sharing the insight.
@farzonl the newly added unit test does fail for Win7, despite the fact that the APIs exists since Win2K. Cross-plat will be a separate conversation and the direction TBD.